install.md

```diff
@@ -198,3 +198,96 @@ If you are using OSCAR in the Windows Subsystem for Linux, you will require a br
 in your subsystem. This can be a probem as the default subsystem is Ubuntu and Ubuntu 
 installs browsers via snap which is disabled for subsystems. To circumvent this problem, 
 please see [how to install browsers via deb](https://www.omgubuntu.co.uk/2022/04/how-to-install-firefox-deb-apt-ubuntu-22-04).
+
+
+## System-wide installation of Oscar
+
+It is possible to install Oscar and its cornerstones only once on your system,
+and to let several people use this installation.
+For that, the following `bash` scripts can be used.
+
+<div class="clickdesc">
+
+<details>
+<summary>
+Install Oscar system-wide,
+or update the system-wide installation of Oscar when a new version is available.
+</summary>
+Enter the following commands into a file (for example <code>oscar_systemwide_install</code>),
+adjust the paths for the variables <code>julia_for_oscar</code> and <code>central_depot</code>,
+and then run the script in a terminal (with administrator rights).
+{% highlight bash %}
+#!/bin/bash
+
+# Specify the intended version of Julia.
+julia_for_oscar=/users/oscar/julia-1.8.5/bin/julia
+
+# Specify the intended location of the central Oscar installation.
+central_depot=/users/oscar/JULIA_DEPOT
+
+# We will store the necessary artifacts in the
+# "architecture dependent location" in Julia's default DEPOT_PATH.
+# First determine this path and create the directory if necessary.
+system_depot=$(${julia_for_oscar} --startup-file=no -e 'println(DEPOT_PATH[2])')
+if ! test -d "${system_depot}/artifacts"; then
+  mkdir -p "${system_depot}/artifacts"
+fi
```

Comment:
```suggestion
mkdir -p "${system_depot}/artifacts"
```
I think `mkdir -p` already is idempotent, so there should be no need for a separate `test`?